exchanges/huobi/huobi_test.go

```diff
@@ -533,16 +1657,13 @@ func TestCancelExchangeOrder(t *testing.T) {
 }
 
 func TestCancelAllExchangeOrders(t *testing.T) {
-	if areTestAPIKeysSet() && !canManipulateRealOrders {
-		t.Skip("API keys set, canManipulateRealOrders false, skipping test")
-	}
-
```

Comment:
probably reinstate this

Reply:
probably rewrite this

Reply:
![image](https://user-images.githubusercontent.com/31364354/101541871-157a3280-39f6-11eb-823b-2b8a25aef529.png)
Its doing the check below. Do you want me to remove that and add this instead?